zest/releaser/baserelease.py

```diff
@@ -1,6 +1,9 @@
 """Provide a base for the three releasers"""
 
+from __future__ import unicode_literals
+
```

Comment:
Note: I normally don't put a blank line in here as it are all stdlib imports.

But... you don't need to change it, I'm planning to run an automatic tool over the imports anyway as there are multiple places where they're not properly sorted.